### ISSUE TITLE
Write a custom loss function

### ISSUE THREAD

**Issue by jsta** [jsta/lstm-pytorch#1]
https://spandan-madan.github.io/A-Collection-of-important-tasks-in-pytorch/#SECTION-5---CUSTOM-LOSS-FUNCTIONS
